README.md

```diff
@@ -37,23 +37,24 @@ By utilizing the combination of source views, computed views, and other relevant
-| mod-fqm-manager.entity-type-cache-timeout-seconds | 3600          | Cache duration for entity type definitions                |
-| server.port                                       | 8081          | Server port                                               |
-| QUERY_RETENTION_DURATION                          | 3h            | Older queries get deleted                                 |
-| task.execution.pool.core-size                     | 9             | Core number of concurrent async tasks                     |
-| task.execution.pool.max-size                      | 10            | Max number of concurrent async tasks                      |
-| task.execution.pool.queue-capacity                | 1000          | Size of the task queue                                    |
+| Name                                              | Default Value | Description                                       |
+|---------------------------------------------------|---------------|---------------------------------------------------|
+| DB_HOST                                           | localhost     | Postgres hostname                                 |
+| DB_PORT                                           | 5432          | Postgres port                                     |
+| DB_HOST_READER                                    | localhost     | Postgres hostname                                 |
+| DB_PORT_READER                                    | 5432          | Postgres port                                     |
+| DB_USERNAME                                       | postgres      | Postgres username                                 |
+| DB_PASSWORD                                       | postgres      | Postgres password                                 |
+| DB_DATABASE                                       | postgres      | Postgres database name                            |
+| MAX_QUERY_SIZE                                    | 1250000       | max result count per query                        |
+| mod-fqm-manager.permissions-cache-timeout-seconds | 60            | Cache duration for user permissions               |
+| mod-fqm-manager.entity-type-cache-timeout-seconds | 3600          | Cache duration for entity type definitions        |
+| server.port                                       | 8081          | Server port                                       |
+| QUERY_RETENTION_DURATION                          | 3h            | Older queries get deleted                         |
+| task.execution.pool.core-size                     | 9             | Core number of concurrent async tasks             |
+| task.execution.pool.max-size                      | 10            | Max number of concurrent async tasks              |
+| task.execution.pool.queue-capacity                | 1000          | Size of the task queue                            |
+| IS_EUREKA                                         | FALSE         | Specifies if environment is configured for EUREKA |
```

Comment:
Nit: The rest are alphabetized, so this should go between the DB settings and MAX_QUERY_SIZE
```suggestion
| Name                                              | Default Value | Description                                       |
|---------------------------------------------------|---------------|---------------------------------------------------|
| DB_HOST                                           | localhost     | Postgres hostname                                 |
| DB_PORT                                           | 5432          | Postgres port                                     |
| DB_HOST_READER                                    | localhost     | Postgres hostname                                 |
| DB_PORT_READER                                    | 5432          | Postgres port                                     |
| DB_USERNAME                                       | postgres      | Postgres username                                 |
| DB_PASSWORD                                       | postgres      | Postgres password                                 |
| DB_DATABASE                                       | postgres      | Postgres database name                            |
| IS_EUREKA                                         | FALSE         | Specifies if environment is configured for EUREKA |
| MAX_QUERY_SIZE                                    | 1250000       | max result count per query                        |
| mod-fqm-manager.permissions-cache-timeout-seconds | 60            | Cache duration for user permissions               |
| mod-fqm-manager.entity-type-cache-timeout-seconds | 3600          | Cache duration for entity type definitions        |
| server.port                                       | 8081          | Server port                                       |
| QUERY_RETENTION_DURATION                          | 3h            | Older queries get deleted                         |
| task.execution.pool.core-size                     | 9             | Core number of concurrent async tasks             |
| task.execution.pool.max-size                      | 10            | Max number of concurrent async tasks              |
| task.execution.pool.queue-capacity                | 1000          | Size of the task queue                            |
```